prow/spyglass/lenses/buildlog/lens_test.go

```diff
@@ -193,3 +195,15 @@ func TestGroupLines(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkHighlightLines(b *testing.B) {
```

Comment:
I think I'd rather this either be self-contained or removed.

Reply:
I am fine adding the file to this repo but it has par design 170k lines. I mostly wanted to provide a way to reproduce my results. Do you prefer adding the file or removing the benchmark?

Reply:
We scan the regexes line by line, so you should really only need a very small (single digit?) of representative lines to still get a meaningful benchmark result. Those can then just be inlined here.

Reply:
Higher variance is expected - you were previously effectively averaging the result of doing this 21,250 times for each sample, which would smooth out the variability.